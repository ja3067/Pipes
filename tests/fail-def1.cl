if False:
	x = 3

print(x)
