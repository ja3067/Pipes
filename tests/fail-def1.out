RuntimeError: name 'x' is not defined
